src/libs/Launcher.js

```diff
@@ -159,42 +160,30 @@ export default class Launcher {
   }
 
   /**
-   * Ensure that the account and the destination folder get the name corresponding to sourceAccountIdentifier
+   * Ensures that that the account has the proper account name which is the sourceAccountIdentifier fetched by the konnector
+   *
+   * @param {import('cozy-client/types/types').IOCozyAccount} account - cozy account
+   * @returns {Promise<import('cozy-client/types/types').IOCozyAccount>}
    */
-  async ensureAccountNameAndFolder() {
-    const { trigger, account, client } = this.getStartContext()
-
-    const firstRun = !get(account, 'auth.accountName')
-    if (!firstRun) {
-      return
-    }
-
+  async ensureAccountName(account) {
+    const { client, konnector } = this.getStartContext()
     const { sourceAccountIdentifier } = this.getUserData() || {}
-    // @ts-ignore
-    const folderId = trigger.message.folder_to_save
-
-    if (!account._id) {
-      throw new Error('ensureAccountNameAndFolder: no account to check')
+    if (!account?._id) {
```

Comment:
means that `account` is not mandatory so? If yes, please update the jsdoc

Reply:
The next line is a `throw`, so it is mandatory.

Reply:
Woups I answered too fast, you are talking about `account`, not `account._id` 🤦

Reply:
I removed the ? . The account is mandatory